src/bindings/python/CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ set(INTERFACE_SRCS
   ../interface/growtree.i
   ../interface/indices.i
   ../interface/imports.i
+  ../interface/precision.i
```

Comment:
No need to expose precision to python bindings, only needed when someone will run routines from the python front end. Can delete this line and file.